dataflows-cloud/helsinki-transit/connector/README.md

```diff
@@ -0,0 +1,41 @@
+# Connector for Helsinki Transit Dataflow
+
+Use mqtt-source connector to read the Helkinki Transit live data feed.
```

Comment:
Start the mqtt-source connector to read the Helkinki Transit live data feed on InfinyOn Cloud.